src/azure-cli/HISTORY.rst

```diff
@@ -83,6 +83,11 @@ Release History
 
 * GA `az eventgrid system-topic` (#20531)
 
+**IoT Central**
+
+* `az iot central app create --mi-system-assigned`: New `--mi-system-assigned` parameter for app create to assign a system-assigned managed identity to the app (#20448)
+
+
```

Comment:
Please do not add the `HISTORY.rst` manually. The history notes will be automatically generated by PR title and description. 
Please refer to this doc [submitting-pull-requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests) for details

Reply:
Fixed. Updated title and description.